.circleci/config.yml

```diff
@@ -1693,3 +1706,20 @@ workflows:
           - collector-support
         requires:
           - upload-modules
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      *reusejobs
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only: jv/ROX-9961-create-nightly-collector-circleci-job
```

Comment:
- [ ] Change to master before merging

Reply:
Probably not something to address in this PR, but since we had issues a few months back with drivers not being built for legacy versions of collector it might be worth it to add a few tags alongside master here for future proving.